lib/screens/util.ex

```diff
@@ -148,4 +148,18 @@ defmodule Screens.Util do
       Time.compare(start_time, t) in [:lt, :eq] or Time.compare(stop_time, t) == :gt
     end
   end
+
+  @doc """
+  Takes route id, returns route type. 
+  Not the favorite way of getting route type, but useful for property testing.
+  """
+  @spec route_type_from_id(String.t()) :: atom()
+  def route_type_from_id("Green-" <> _), do: :light_rail
```

Comment:
I believe `Green` without a branch letter is also valid. So you may want to remove the `-` here to catch those as well.

```suggestion
  def route_type_from_id("Green" <> _), do: :light_rail
```